undertow/src/main/java/org/wildfly/extension/undertow/UndertowTransformers.java

```diff
@@ -112,6 +115,13 @@ private static void registerTransformers_EAP_7_2_0(ResourceTransformationDescrip
                     .addRejectCheck(RejectAttributeChecker.DEFINED, ENABLE_JASPI, INTEGRATED_JASPI)
                     .setDiscard(DiscardAttributeChecker.ALWAYS, ENABLE_JASPI, INTEGRATED_JASPI) // Discard so we don't send over the defaults.
                 .end();
+
+        ResourceTransformationDescriptionBuilder builder = subsystemBuilder.addChildResource(UndertowExtension.PATH_FILTERS)
+                .addChildResource(PathElement.pathElement(Constants.MOD_CLUSTER));
+
+        builder.discardChildResource(NoAffinityResourceDefinition.PATH);
```

Comment:
The legacy behavior corresponds to SingleAffinityResourceDefinition.PATH, thus I think we want to use:
builder.rejectChildResource(NoAffinityResourceDefinition.PATH)

Reply:
Oops, fixed.